res/lang/advertisement/advertisement_pl.txt

```diff
@@ -29,6 +29,7 @@ AD_STILL_X_DAYS       = ciągle %1 dni
 AD_CHANNEL_IMAGE_TOO_LOW = Wizerunek za niski. %CHANNELIMAGE%% < %IMAGE%%.
 AD_PLEASE_GENRE_X        = Proszę o gatunek %GENRE%
 AD_PLEASE_FLAG           = Proszę %FLAG%
+#AD_PLEASE_NOT_FLAG       = Proszę nie %FLAG%
```

Comment:
Warum auskommentiert?

Reply:
@pauldim : would this be correct for a flag or flags (like live/x-rated/culture) not allowed for an ad contract?